spec/jobs/publish_to_twitter_job_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe PublishToTwitterJob, :type => :job do
+  subject { PublishToTwitterJob.new }
+
+  describe "#perform" do
+    let(:artist) { User.new(name: 'joe') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.